src/config/syscheck-config.c

```diff
@@ -1223,21 +1235,15 @@ static void parse_synchronization(syscheck_config * syscheck, XML_NODE node) {
                 syscheck->enable_registry_synchronization = r;
             }
 #endif
-        } else if (strcmp(node[i]->element, xml_min_sync_interval) == 0) {
-            long interval = w_parse_time(node[i]->content);
-
-            if (interval < 0) {
-                mwarn(XML_VALUEERR, node[i]->element, node[i]->content);
-            } else {
-                syscheck->min_sync_interval = (uint32_t) interval;
-            }
         } else {
             mwarn(XML_INVELEM, node[i]->element);
         }
     }
 
-    if (syscheck->min_sync_interval >= syscheck->sync_interval) {
-        mwarn("Sync min_interval value higher than interval. Some synchronization turns can be skipped.");
+    if (syscheck->sync_max_interval < syscheck->sync_interval) {
+        syscheck->sync_max_interval = syscheck->sync_interval;
```

Comment:
If this is a warning, why have you assigned a value to sync_max_interval?

Reply:
It is a warning to alert the user that something in the configuration is wrong. However, we can set the closest value to the user choice and continue with the program.